src/main/java/org/isf/medicalstock/manager/MovStockInsertingManager.java

```diff
@@ -418,4 +418,17 @@ private List<Movement> prepareDishargingMovement(Movement movement, boolean chec
 			return dischargeMovement;
 		}
 	}
+	
+	/**
+	 * Save the specified {@link Movement}
+	 *
+	 * @param movement - the movement to store.
+	 * @return the stored {@link Movement} object.
+	 * @throws OHServiceException.
+	 */
+	@Transactional(rollbackFor = OHServiceException.class)
+	public Movement storeMovement(Movement movement) throws OHServiceException {
+		return ioOperations.newMovement(movement);
```

Comment:
bypassing `validateMovement`?

Reply:
Also, maybe newMovement in org/isf/medicalstock/service/MedicalStockIoOperations.java should be protected (and not public) to avoid this direct call from other packages

Reply:
I have remove the method